haystack/environment.py

```diff
@@ -104,13 +104,14 @@ def collect_static_system_specs() -> Dict[str, Any]:
     except ImportError:
         specs["libraries.transformers"] = False
 
+    has_mps = hasattr(torch.backends, "mps") and torch.backends.mps.is_available()
```

Comment:
My bad, didn't notice until the tests failed - this lline should go below `torch_import.check()` at line 109